integration/clean/README.md

```diff
@@ -1,18 +1,42 @@
-**What does the cleaner do?**
-
-###Cleaner cleans out old ami (ami older than 60 days)
-
-The cleaner first searches for ami names (these are the ami created by the pipeline for use int he integration tests)
-1. cloudwatch-agent-integration-test*
-
-Then checks to see if the creation date is greater than 60 days. (The aws sdk v2 gives creation date as a pointer to string. To convert to golang time we use the aws smithy go time. This allows us to compare to 60 days in past time)
-
-If the ami is older than 60 days old then we delete the ami
-
-###Cleans dedicated hosts for mac
-
-The cleaner first searches for dedicated host tag Name:IntegrationTestMacDedicatedHost
-
-Then checks to see if the creation date is greater than 26 hours and host status is available
-
-Delete is true
\ No newline at end of file
+## 1. What does the cleaner do?
+The cleaner will clean the AWS resources which are created from Integration Test.
+* CloudWatch Agent Integration Test
+
+### 1.1 Cleans AMI 
+The cleaner first searches for AMI' name that begins with `cloudwatch-agent-integration-test`
+The cleaner will delete the ami is older than 30 days old as default. However, you can change 
+the 30 days by using variable `keep` as [the below example](README.md#2-how-to-use-the-script).
+
+### 1.2 Cleans dedicated hosts for mac
+The cleaner first searches for dedicated host's tag `Name:IntegrationTestMacDedicatedHost`. 
+Then checks to see if the creation date is greater than 30 days as default and host status is 
+available. However, you can change the 30 days by using variable `keep` as [the below example](README.md#2-how-to-use-the-script).
+
+### 1.3 Cleans SSM Parameter Store
+The cleaner first searches for SSM Parameter Stores' name that begins with `AmazonCloudWatch`
+Then checks to see if the creation date is greater than 30 days as default. However, you can 
+change the 30 days by using variable `keep` as [the below example](README.md#2-how-to-use-the-script).
+
+## 2. How to use the script?
+By running the below command **with or without replacing these variables**:
+* **keep:** days to keep your resources (i.e 5 which is deleting resources after existing for 5 days)
+* **clean:** resources need to be cleaned (only support dedicated_host, ssm, ami)
```

Comment:
those are the only clean scripts we have so why are we calling that it is only supported for these?

Reply:
Nice catch on wording. I should have change to `only have` instead of `only support`; support is more towards customer instead of developers.